src/actions/motion.ts

```diff
@@ -576,9 +569,23 @@ class MoveToColumn extends BaseMovement {
   }
 }
 
+interface RepeatableMovement extends BaseMovement {
+  isRepeat: boolean;
+}
+
+export function createRepeatMovement(
```

Comment:
Do you mean that instead of extending `BaseMovement`, `MoveFindForward` (for example) should extend a new class `RepeatableMovement` whose constructor takes two optional parameters (`isRepeat` and `keysPressed`)?

Reply:
Actually, can you add `isRepeat` to `BaseMovement`? No need to create a new class.

Reply:
Ok. Currently `BaseMovement` has no constructor, so should I create one, or keep the `createRepeatMovement` helper function?

Reply:
Constructor. Never been a fan of factory methods when that's the purpose of a constructor.